tests/samples/DashboardSample.js

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+export const LABELS = {
+  noScenario: {
+    title: 'ut_noScenario_title',
+    label: 'ut_noScenario_label',
+  },
+  noDashboard: {
+    label: 'ut_noDashboard_label',
+  },
+  noRun: {
+    label: 'ut_noRun_label',
+  },
+  inProgress: {
+    label: 'ut_InProgress_label',
```

Comment:
just a question: all labels begin with a lowercase letter and this one begins with a capitalized _I,_ is it on purpose ? :slightly_smiling_face: